docs/ci-cd-environment/sem-version-managing-language-versions-on-linux.md

```diff
@@ -1,5 +1,3 @@
- # sem-version: Managing Language Versions on Linux
-
 The `sem-version` utility in Linux based virtual machines is used for changing
```

Comment:
is used `to change` versions of programming languages
 Sounds better in my opinion. Lets ask for a third pair of eyes.

Reply:
But I didn't change that sentence at all. I just removed that one line that is currently at the top of the page.